editions/tw5.com/tiddlers/macros/TabsMacro.tid

```diff
@@ -29,3 +29,7 @@ Within the template, the title of the selected tab is available in the <<.var cu
 The <<.vlink currentTiddler>> variable is not affected by the <<.var tabs>> macro.
 
 <<.macro-examples "tabs">>
+
+!! Preserving Tab Contents
+
+<<.from-version "5.1.14">> To preserve tab contents, e.g. iframes or embedded content, set the field ''retain-tab'' to `yes` at a tab tiddler. This will prevent reloading the content when switching tabs.
```

Comment:
may be: `embedded content (eg: active music or videos streams)...` would make it clearer. .. imo for images it's not needed but for everyting, where the active "state" would be destroyed.